src/lucky_env.cr

```diff
@@ -23,4 +23,18 @@ module LuckyEnv
       load(file_path)
     end
   end
+
+  def self.name
```

Comment:
I don't particularly like calling it `name` when it's on the root namespace, but couldn't really think of anything better. I think `name` was fine, when it was nested under `Lucky::Env` though.

If anyone have a suggestion, I'd love to hear it. 🙂

Reply:
Sounds good. I addressed this in https://github.com/luckyframework/lucky_env/pull/13/commits/6ad252bde6eb8b38b47685dfa1ff31531af693be. 🙂